tasks/unit-tests/components_tests.py

```diff
@@ -0,0 +1,69 @@
+import os
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Can you take ownership of this file and of `test-data/components_src`?

Reply:
Done.